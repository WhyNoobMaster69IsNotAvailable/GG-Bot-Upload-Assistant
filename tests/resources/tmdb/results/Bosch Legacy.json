{
    "page": 1,
    "results": [
        {
            "backdrop_path": "/wVGiwvE5CtlVPrP0oEO3W70ft6D.jpg",
            "first_air_date": "2022-05-05",
            "genre_ids": [
                80,
                18
            ],
            "id": 153657,
            "name": "Bosch: Legacy",
            "origin_country": [
                "US"
            ],
            "original_language": "en",
            "original_name": "Bosch: Legacy",
            "overview": "Bosch is now making a living as a private investigator two years after he quit the LAPD and finds himself working with one time enemy and top-notch attorney Honey “Money” Chandler. Meanwhile, Bosch's daughter Maddie is venturing into the world of the LAPD.",
            "popularity": 14.973,
            "poster_path": "/9Z7L7wSB6nLwWk7coUgm1ytQbOB.jpg",
            "vote_average": 8.4,
            "vote_count": 30
        }
    ],
    "total_pages": 1,
    "total_results": 1
}
