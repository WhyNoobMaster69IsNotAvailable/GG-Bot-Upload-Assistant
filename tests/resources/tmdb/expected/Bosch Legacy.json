{
    "tmdb": "153657",
    "imdb": "0",
    "tvmaze": "0",
    "possible_matches": [
        {
            "result_num": 1,
            "title": "Bosch: Legacy",
            "content_type": "tv",
            "tmdb_id": 153657,
            "release_date": "2022-05-05",
            "language": "en",
            "overview": "Bosch is now making a living as a private investigator two years after he quit the LAPD and finds himself working with one time enemy and top-notch attorney Honey “Money” Chandler. Meanwhile, Bosch's daughter Maddie is venturing into the world of the LAPD."
        }
    ]
}